examples/platforms/efr32/radio.c

```diff
@@ -770,15 +773,19 @@ void efr32RadioProcess(otInstance *aInstance)
     CORE_EXIT_CRITICAL();
 }
 
-void RAILCb_RfReady(void)
+void RAILCb_RxAckTimeout(void)
 {
+    if (sIsAckRequested)
```

Comment:
Seems odd to be waiting for an ACK when the packet does not request one.  How about using the following sequence when no ack is requested?

```
    RAIL_TxOptions_t txOption;
    txOption.waitForAck = false;
    RAIL_Status_t status = RAIL_TxStartWithOptions(0, &txOption, NULL, NULL);
```